recipe/meta.yaml

```
@@ -16,10 +16,9 @@ source:
 
 build:
   number: {{ build }}
-  skip: true  # [(win and py==310) or (cuda_compiler_version in (undefined, 'None') and (linux64 or win)) or (cuda_compiler_version != '10.2')]
+  skip: true  # [(cuda_compiler or "None") != "None"]
```

Oops 🤦‍♂️ Sorry this should be `==`s

Please also re-render after this change

Should fix the latest CI errors

```suggestion
  skip: true  # [(cuda_compiler or "None") == "None"]
```